lib/config.js

```diff
@@ -107,6 +107,7 @@ config.workerWarmUpDelayMS = 1000;
 config.workerUseQueue = true;
 config.workerOverloadDelayMS = 10000;
 config.groupName = 'local'; // used for load reporting
+config.imageProcessingDir = './image_processing';
```

Comment:
It comes down to your personal preference. I can see pros and cons here. Pro is development and debugging.

Reply:
I'd rather use a temporary directory here, possibly coupled with appropriate `debug(...)` logs that we can turn on to see where the file is stored if we ever need to debug this. I'm mostly aiming to avoid the proliferation of config options.

Reply:
Done in part 2.